unittests/test_mapper.py

```diff
@@ -9,16 +9,15 @@
 from bomf.model import Bo4eTyp
 
 
-class _NotImplementedBo4eDataSetMixin:
+class _NotImplementedBo4eDataSetMixin(BaseModel):
```

Comment:
```suggestion
class _NotImplementedBo4eDataSetMixin:
```
brauchst du das base-model hier? denn es ist ja ein mixin